mantidproject/mantid
# Clean up LoadVesuvio

Fixes #14910

This issue has included:
- Removing 9 period case
- Improving validation
- Removing unused map options for 2 and 3 period cases
- Improving commenting
- Remove invalid tests and Adding new failure cases
# To Test
- Ensure tests pass and the changes to the tests look sensible
- Try to run `LoadVesuvio` with invalid values including incorrect scattering vs. modes of operation. Ensure that for these cases LoadVesuvio does not run
